sass/patternfly-react/_CardMatchHeight.scss

```diff
@@ -0,0 +1,17 @@
+.card-pf-match-height {
```

Comment:
Shouldn't it be added to all cards?

Reply:
I think we should separate the two types of cards, giving consumers more freedom with the HeightMatching Grid (e.g. 2 match-height cards and 1 fixed height card)